posthog/models/team/team.py

```diff
@@ -158,20 +158,27 @@ def get_effective_membership_level_for_parent_membership(
             or not self.access_control
         ):
             return requesting_parent_membership.level
-        from ee.models import ExplicitTeamMembership
         from posthog.models.organization import OrganizationMembership
 
         try:
-            return (
-                requesting_parent_membership.explicit_team_memberships.only("parent_membership", "level")
-                .get(team=self)
-                .effective_level
-            )
-        except ExplicitTeamMembership.DoesNotExist:
+            from ee.models import ExplicitTeamMembership
```

Comment:
same q. 

I appreciate that it might be better to do these in a follow up PR.

Just pointing these out here so we don't miss these